src/doc/python_scripting/quickrecipes.rst

```diff
@@ -45,7 +45,92 @@ To make sure that the plot gets drawn, call the DrawPlots function.
     :end-before: # getting something on the screen }
     :dedent: 4
 
-Handling Command line Arguments
+Using VisIt with the system Python
+----------------------------------
+
+There are situations where you may want to import the VisIt_ module into the system Python.
+Some common use cases are using VisIt_ as part of a larger Python workflow or when you need to use a Python module that VisIt_'s Python does not include.
+You should always try to use VisIt_'s Python interpreter directly, since importing VisIt's Python module may not always work.
+
+When importing the VisIt_ module into the system Python there are, in general, three things you must do.
```

Comment:
Should python version information be included here somewhere?
I mean, doesn't the system python version need to match VisIt's python version in at least major and minor?